Add information about lack of licensed/unlicensed interoperability (#1679)
meshtastic/meshtastic: docs/about/faq.mdx

- Instructions for Enabling ham License from the Python CLI can be found [here](/docs/software/python/cli/usage#ham-radio-support).
      `,
    },

>     {
>       title: "Why aren’t licensed and unlicensed modes interoperable? Is this a planned feature?",
>       content: `
> - Amateur radio regulations vary widely around the world; many countries have specific and often differing regulations against the use of encryption over amateur radio. 
> - Additionally, many countries have regulations preventing licensed operators from relaying third party messages (in this case, the messages of unlicensed stations) or the relaying of licensed stations’ messages without an accompanying call sign. 
> - This makes interoperability impossible at worst and unfeasible at best; as a result interoperability between these modes is not a planned feature.`,
>     },

Reviewer: ```suggestion
    {
      title: "Why aren’t licensed and unlicensed modes interoperable? Is this a planned feature?",
      content: `
Amateur radio regulations differ significantly across countries, with many prohibiting the use of encryption. Additionally, regulations in some countries restrict licensed operators from relaying messages from unlicensed stations or transmitting messages from licensed stations without including a call sign. These restrictions make interoperability highly impractical, if not impossible. Consequently, interoperability between these modes is not a planned feature.`,
    },
```

I don’t think a bullet list is necessary here. I’ve suggested a few edits to tighten the text and improve clarity.
